tests/network/localnet/conftest.py

```diff
@@ -60,60 +69,189 @@ def cudn_localnet(
     namespace_localnet_2: Namespace,
 ) -> Generator[libcudn.ClusterUserDefinedNetwork]:
     with localnet_cudn(
-        name=NETWORK_NAME, match_labels=LOCALNET_TEST_LABEL, vlan_id=vlan_id, physical_network_name=NETWORK_NAME
+        name=LOCALNET_BR_EX_NETWORK,
+        match_labels=LOCALNET_TEST_LABEL,
+        vlan_id=vlan_id,
+        physical_network_name=LOCALNET_BR_EX_NETWORK,
     ) as cudn:
         cudn.wait_for_status_success()
         yield cudn
 
 
+@pytest.fixture(scope="module")
+def ipv4_localnet_address_pool() -> Generator[str]:
+    return (f"10.0.0.{host_value}/24" for host_value in range(1, 254))
```

Comment:
It's not a must, but I think that in order to conform to the common convention, the range should start in 2 and not in 1. `x.y.z.1` is usually used for the default gateway.

Reply:
The common convention in our tests is to use host ID 1 for the first VM :) But I don't mind changing it, it is a good practice to leave 1 available for the default gateway, even though we don't use it in this network we define for our tests.

Reply:
Yeah, you make a good point, and for some reason I haven't noticed so far that we use IP ranges starting from 1.
So again - I'll leave it to you to decide whether to start it from 2 or leave it as it is.
Anyway you can resolve this comment.